cpp/include/raft/random/multi_variable_gaussian.cuh

```diff
@@ -59,6 +59,81 @@ class multi_variable_gaussian : public detail::multi_variable_gaussian_impl<T> {
   ~multi_variable_gaussian() { deinit(); }
 };  // end of multi_variable_gaussian
 
+/**
+ * @brief Matrix decomposition method for `compute_multi_variable_gaussian` to use.
+ *
+ * `compute_multi_variable_gaussian` can use any of the following methods.
+ *
+ * - `CHOLESKY`: Cholesky decomposition
+ * - `JACOBI`:
```

Comment:
Is this incomplete?

Reply:
It was!  Thanks for pointing this out! : - )  It took me a while to dig out the cuSOLVER documentation explaining the difference between the two algorithms -- I had guessed that one was Jacobi plane rotations, but needed confirmation, and LAPACK doesn't expose these.